src/doc/dev_manual/SiteReliabilityEngineering.rst

```diff
@@ -251,13 +251,18 @@ Active SRE Issues Repo
 `issues-only repository <https://github.com/visit-dav/live-customer-response/issues>`_
 within the `visit-dav GitHub organization <https://github.com/visit-dav>`_. 
 For each new inquiry, the primary will file an issue ticket and assign themselves.
-When the inquiry is :ref:`resolved <sre_response_vs_resolution>`, the associated
-issue is closed. The primary will endeavor to capture all relevant information and
+
+The primary will endeavor to capture all relevant information and
 communications in this issue. The use of GitHub issues for this purpose has a number
 of advantages over other options such as email including better search/browse as well
 as support for attachments. For this reason, a number of steps were taken to integrate
 the ``visit-users@elist.ornl.gov`` email list with this issues-only repository.
 
+The SRE issue shall be closed when the associated inquiry is
+:ref:`resolved <sre_response_vs_resolution>`. Or, it will be closed and labeled
+``wontfix`` after 21 days pass since the user last engaged in any conversation with
+VisIt developers to reach a resolution.
```

Comment:
Can you change "VisIt" to "VisIt_". Sorry about missing that.

Reply:
Whoops. Yeah, I just cut-n-pasted what you had and did read it that closely 😋 . I fixed that in two places actually